GoogleTestAdapter/VsPackage.TAfGT/Resources/VSPackage.resx

```diff
@@ -58,14 +117,24 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="110" xml:space="preserve">
     <value>Test Adapter for Google Test</value>
   </data>
   <data name="112" xml:space="preserve">
     <value>Enables Visual Studio's testing tools with unit tests written for Google Test.  The use terms and Third Party Notices are available in the extension installation directory.</value>
   </data>
+  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="400" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\..\Resources\Icons\Icon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="501" xml:space="preserve">
```

Comment:
Could you explain to me how this works?

Reply:
The name (number) is used in GoogleTestExtensionOptionsPage.cs:26 to specify localized lookups for the category and page names in the options window:

    [ProvideOptionPage(typeof(GeneralOptionsDialogPage), OptionsCategoryName, "General", 110, 501, true)]

Parameters 2 and 3 specify non-localized canonical names for the category and page names, while parameters 3 and 4 specify resource ids for the corresponding localized versions. `ProvideOptionPage` only allows `short`s in these positions, so we are forced to use numbers.

Reply:
Thanks for the explanation.